packages/link/src/spectrum-link.css

```diff
@@ -39,18 +39,19 @@ a:active {
     --mod-link-text-color: var(--mod-link-text-color-active, var(--mod-link-text-color-primary-active, var(--spectrum-accent-content-color-down)));
 }
 
-a:focus-visible {
+:host([quiet]) a {
+    -webkit-text-decoration: none;
+    text-decoration: none;
+}
+
+:host a:focus-visible,
+:host([quiet]) a:focus-visible {
```

Comment:
This update looks great, thank you so much for tackling this! Once the package.json change is reverted, can you run `yarn changeset` and document the update for our Changelogs?